src/colp/components/agency_usetype_report.py

```diff
@@ -5,8 +5,11 @@
 
 
 class CountRecordsReport(ABC):
+    def __init__(self, data) -> None:
+        self.content_to_display = True if data is not None and not data.empty else False
```

Comment:
nit: i think its maybe easier to read to avoid double nots, could be 
```
self.content_to_display = False if data is None or data.empty else True
```
What is the scenario that this is empty and not null?

Reply:
One of the potential errors here is that the table on the colp side is created but not populated, in which case it is read in as an empty dataframe